src/system.rs

```diff
@@ -58,10 +69,10 @@ mod read_dt_props {
     use std::fs::read;
     use std::str::from_utf8;
 
-    const DT_CHOSEN: &str = "/sys/firmware/devicetree/base/chosen/";
+    const DT_BASE: &str = "/sys/firmware/devicetree/base/";
 
     pub fn read_dt_property(path: &str) -> String {
```

Comment:
As you are already touching the `read_dt_property` functions, this is would be the perfect opportunity to replace the panicking behavior of these functions and their dependent callers with proper error handling.

Reply:
Now `src/system.rs` is officially `.unwrap()` free 🎉.

Reply:
Nice! :tada: